python/lsst/ts/xml/data/sal_interfaces/Electrometer/Electrometer_Commands.xml

```diff
@@ -91,10 +91,10 @@
   <SALCommand>
     <Subsystem>Electrometer</Subsystem>
     <EFDB_Topic>Electrometer_command_startScanDt</EFDB_Topic>
-    <Description>Start storing readings inside the electrometer buffer for a fixed time, this command will change detailedState from NotReadingState to SetDurationReadingState and to stop storing data into the buffer and publish the LFO there are 2 possibilities. Reading during scanDuration time input in the command or sending a stopScan command</Description>
+    <Description>Start storing readings inside the electrometer buffer for a fixed time, this command will change detailedState from NotReadingState to SetDurationReadingState and to stop storing data into the buffer and publish the LFA there are 2 possibilities. Reading during scanDuration time input in the command or sending a stopScan command.</Description>
```

Comment:
I think there is a typo here "there are 2 possibilities" .. and the last sentence is confusing. I think pieces of phrases just got messed up here...

Reply:
I think it makes more sense now.